homework/hackyourtemperature/server.js

```diff
@@ -0,0 +1,16 @@
+import Express from "express";
```

Comment:
```suggestion
import express from "express";
```

Usually we stick to `camelCase` when using variables in JS. Variables with `PascalCase` usually indicate a class or something we instantiate i.e. `const collin = new Person()`